monero-outreach-docs/translations/pt-br/quickfacts/QuickFacts_pt-br.md

```diff
@@ -0,0 +1,125 @@
+# MONERO
+
+Monero é uma moeda digital segura, privada, não-rastreável que está disponível para todos. As suas caracterísitcas o tornam uma moeda fungível.
+
+# FATOS RÁPIDOS
+
+Usuários podem gastar com segurança, ninguém consegue ver seu saldo ou rastrear suas atividades sem sua permissão.
+
+Revisado em 1/2/2019 - [Printer Friendly Version](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_PrinterFriendly.pdf)
+
+- [Versão colorida](http://www.monerooutreach.org/pubs/2018/QuickFacts/QuickFacts_pt.pdf)
+
+[getmonero.org](https://getmonero.org/)
+
+
+## Princípios Fundamentais
+
+#### DESCENTRALIZAÇÃO
+
+Sendo um _[projeto código aberto](https://github.com/monero-project/monero)_ liderado e _[financiado](https://forum.getmonero.org/8/funding-required)_ por um _[time decentralized](https://getmonero.org/community/team/)_ de desenvolvedores e membros da comunidade, o mesmo não pode ser censurado. A maioria dos contribuidores são voluntários e a comunidade está espalhada por todo o globo.

```

Comment:
Would be good to link "funding required" to the brand-new CCS. Also pinging @lh1008 

```suggestion
Sendo um _[projeto código aberto](https://github.com/monero-project/monero)_ liderado e _[financiado](https://ccs.getmonero.org/funding-required/)_ por um _[time decentralized](https://getmonero.org/community/team/)_ de desenvolvedores e membros da comunidade, o mesmo não pode ser censurado. A maioria dos contribuidores são voluntários e a comunidade está espalhada por todo o globo.
```